frontstage/templates/surveys/surveys.html

```diff
@@ -48,7 +48,7 @@ <h2 class="ons-u-mb-xs ons-u-mt-no ons-u-fs-r ons-u-fw-b"><span>Period covered:<
                 </dd>
                 <dt style="float: left; clear: left;" class="ons-u-vh">Help: </dt>
                 <dd class="ons-u-ml-no ons-u-mb-s">
-                    <a id="create-message-link-{{loop.index}}" href="{{ url_for('surveys_bp.get_surveys_help_page', survey_ref=survey.survey_ref, ru_ref=survey.business_ref) }}">Get help with this survey <span class="ons-u-vh"> about {{ survey.survey_long_name }}</span></a>
+                    <a id="create-message-link-{{loop.index}}" href="{{ url_for('surveys_bp.get_surveys_help_page', survey_ref=survey.survey_ref, ru_ref=survey.business_ref, collection_exercise_id=survey.collection_exercise_id, survey_id=survey.survey_id, period_ref=survey.period_ref) }}">Get help with this survey <span class="ons-u-vh"> about {{ survey.survey_long_name }}</span></a>
                 </dd>
```

Comment:
Fine for this change but we will need to find a better way of providing this to the template from the Python object